src/pages/dev/reference/glossary.mdx

```diff
@@ -0,0 +1,59 @@
+# Axelar Glossary
+
+The purpose of this document is to provide you with a better understanding of the terminology used in the Axelar documentation. 
+
+## Canonical
+
+A canonical token in blockchain is a standardized representation of a digital asset, such as a cryptocurrency or token, accepted as the standard across multiple blockchains in a network. This standardization ensures consistent behavior and enables interoperability between blockchains, allowing cross-chain transactions and interactions within blockchain networks that support various chains. 
+
+In the context of Axelar's Interchain Token Service, a canonical token is an interchain token built from an existing token that is available on multiple chains. The token deployed to other chains will be a [Standardized](#standardized) token, and there can only be one canonical token for every ERC-20.
+
+## Custom
+A custom token in blockchain is a user or organization-created digital asset, such as a cryptocurrency or token, designed to meet specific needs. Unlike standardized tokens, custom tokens are highly customizable, allowing their creators to define unique features, functionalities, and rules. They serve particular use cases within the blockchain ecosystem and offer flexibility.
+
+You can create and deploy a custom ERC-20 token using the Interchain Token Service. Unlike canonical tokens, which are standardized, custom tokens can have unique features and functionality tailored to your specific requirements. It can optionally implement the `IInterchainToken` interface for transfer methods directly on your token.
+
+## Distributor
```

Comment:
This isn't a role that exists on native tokens for example.

Reply:
Updated. Let me know what you think about this change.